src/flyte/_image.py

```diff
@@ -25,6 +28,18 @@
 T = TypeVar("T")
 
 
+@rich.repr.auto
+@dataclass(frozen=True, repr=True)
+class SecretMount:
```

Comment:
I thought we were going to delete this right?

Reply:
deleted it